src/particles/pusher/GetAndSetPosition.H

```diff
@@ -43,8 +45,11 @@ struct GetParticlePosition
      */
     GetParticlePosition (const T_ParTile& a_ptile, int a_offset = 0) noexcept
```

Comment:
It now seems that we create an instance of the `GetParticlePosition` only twice, and call the functor only 3 times. Is it still relevant to keep it now that we go full SoA? Just starting the discussion here.

Reply:
I removed them.